File: app/src/main/java/com/shenghaiyang/oksign/library/Library.java
package com.shenghaiyang.oksign.library;

final class Library {
  final String name;
  final String url;
  final String licenseName;
  final String licenseContent;

  Library(String name, String url, String licenseName, String licenseContent) {
    this.name = name;
    this.url = url;
    this.licenseName = licenseName;
    this.licenseContent = licenseContent;
  }
}

File: app/src/main/java/com/shenghaiyang/oksign/library/Library.kt
package com.shenghaiyang.oksign.library

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Library(
        @SerialName("name")
        val name: String,
        @SerialName("url")
        val url: String,
        @SerialName("licenseName")
        val licenseName: String,
        @SerialName("licenseContent")
        val licenseContent: String
)
